searching/exponential_search.c

```diff
@@ -81,7 +82,7 @@ int main()
 static void test()
 {
     // empty array
-    int64_t arr_empty[] = {};
+    int64_t arr_empty[] = { 0 };
     assert(exponential_search(arr_empty, 0, 10) == -1);
```

Comment:
This might fail

Reply:
It didn't. I tested it on Ubuntu/Gitpod and it worked fine.

Reply:
i see alright!